packages/plugin-rest-client-shared-cache/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ima-plugin-rest-client-shared-cache",
+  "version": "0.0.1",
+  "description": "Generic REST API client plugin for the IMA application framework.",
+  "main": "index.js",
+  "scripts": {
```

Comment:
update scripts to https://github.com/seznam/IMA.js-plugins/blob/master/packages/plugin-self-xss/package.json#L6